lua/starfall/preprocessor.lua

```diff
@@ -121,6 +123,17 @@ SF.Preprocessor = {
 						""
 					}, "\n")
 				end
+
+                local originalCode = files[path]
+				if fdata.obfuscate then
```

Comment:
I think we inherited some sort of mixed-indentation because this looks fine on my end